core/src/test/scala/org/apache/spark/deploy/RPackageUtilsSuite.scala

```diff
@@ -127,7 +127,7 @@ class RPackageUtilsSuite
       RPackageUtils.checkAndBuildRPackage(jar.getAbsolutePath, new BufferPrintStream,
         verbose = true)
       val output = lineBuffer.mkString("\n")
-      assert(output.contains(RPackageUtils.RJarDoc))
+      assert(output.contains("Building R package"))
```

Comment:
This fix seems hacky. Is there a better fix for this one?

Reply:
You are right, this change is not too great.
I reverted the changes to `RPackageUtils.scala` and this file for now.
I will figure out how to get the R packages installed on my local machine and fix this file in a next step.

Reply:
Created https://issues.apache.org/jira/browse/SPARK-48122 to track it separately.